make build TARGET=rhel7 fails; missing bsdtar, gd-devel

Out of the box example fails for me.

```
[root@ose-prod-master-01 src]# git clone  https://github.com/openshift/sti-base.git
[root@ose-prod-master-01 src]# cd sti-base

[root@ose-prod-master-01 sti-base]# make build TARGET=rhel7
SKIP_SQUASH=0 VERSIONS="" OS=rhel7 VERSION= BASE_IMAGE_NAME="base" OPENSHIFT_NAMESPACES="" hack/build.sh
-> Building openshift/base-rhel7 ...
Sending build context to Docker daemon   194 kB
Step 1 : FROM rhel7.2
c453594215e4: Download complete
Status: Downloaded newer image for registry.access.redhat.com/rhel7.2:latest
registry.access.redhat.com/rhel7.2: this image was pulled from a legacy registry.  Important: This registry version will not be supported in future versions of docker.

 ---> c453594215e4
Step 2 : LABEL io.openshift.s2i.scripts-url image:///usr/libexec/s2i io.s2i.scripts-url image:///usr/libexec/s2i BZComponent "openshift-sti-base-docker" Name "openshift3/sti-base" Version "1.0" Releas
e "1" Architecture "x86_64"
 ---> Running in cde0564d6918
 ---> 9ac1f12a9b20
Removing intermediate container cde0564d6918
Step 3 : ENV STI_SCRIPTS_URL image:///usr/libexec/s2i STI_SCRIPTS_PATH /usr/libexec/s2i HOME /opt/app-root/src PATH /opt/app-root/src/bin:/opt/app-root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/us
r/bin:/sbin:/bin
 ---> Running in 75623bd03588
 ---> 3a37f26a827c
Removing intermediate container 75623bd03588
Step 4 : COPY contrib/scl_enable /opt/app-root/etc/scl_enable
 ---> 35adea6b6c57
Removing intermediate container b587d03c56f7
Step 5 : ENV BASH_ENV /opt/app-root/etc/scl_enable ENV /opt/app-root/etc/scl_enable PROMPT_COMMAND ". /opt/app-root/etc/scl_enable"
 ---> Running in 3febe826395f
 ---> 3d6153539eb2
Removing intermediate container 3febe826395f
Step 6 : RUN yum-config-manager --enable rhel-7-server-optional-rpms &&   INSTALL_PKGS="autoconf   automake   bsdtar   findutils   gcc-c++   gd-devel   gdb   gettext   git   libcurl-devel   libxml2-de
vel   libxslt-devel   lsof   make   mariadb-devel   mariadb-libs   openssl-devel   patch   postgresql-devel   procps-ng   scl-utils   sqlite-devel   tar   unzip   wget   which   yum-utils   zlib-devel
" &&   mkdir -p ${HOME}/.pki/nssdb &&   chown -R 1001:0 ${HOME}/.pki &&   yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS &&   rpm -V $INSTALL_PKGS &&   yum clean all -y &&   useradd -u 1001 -r -
g 0 -d ${HOME} -s /sbin/nologin       -c "Default Application User" default &&   chown -R 1001:0 /opt/app-root
 ---> Running in 7ae11564f621
Loaded plugins: ovl, product-id
Loaded plugins: ovl, product-id, search-disabled-repos, subscription-manager
```
**No package bsdtar available.**
```
Package 1:findutils-4.5.11-5.el7.x86_64 already installed and latest version
```
**No package gd-devel available.**
```
Package procps-ng-3.3.10-5.el7_2.x86_64 already installed and latest version
Package yum-utils-1.1.31-34.el7.noarch already installed and latest version
Resolving Dependencies
...
Dependency Updated:
  pcre.x86_64 0:8.32-15.el7_2.1

Complete!
package bsdtar is not installed
package gd-devel is not installed
The command '/bin/sh -c yum-config-manager --enable rhel-7-server-optional-rpms &&   INSTALL_PKGS="autoconf   automake   bsdtar   findutils   gcc-c++   gd-devel   gdb   gettext   git   libcurl-devel   libxml2-devel   libxslt-devel   lsof   make   mariadb-devel   mariadb-libs   openssl-devel   patch   postgresql-devel   procps-ng   scl-utils   sqlite-devel   tar   unzip   wget   which   yum-utils   zlib-devel" &&   mkdir -p ${HOME}/.pki/nssdb &&   chown -R 1001:0 ${HOME}/.pki &&   yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS &&   rpm -V $INSTALL_PKGS &&   yum clean all -y &&   useradd -u 1001 -r -g 0 -d ${HOME} -s /sbin/nologin       -c "Default Application User" default &&   chown -R 1001:0 /opt/app-root' returned a non-zero code: 2
make: *** [build] Error 1
```

--- a/Dockerfile.rhel7
+++ b/Dockerfile.rhel7
@@ -38,7 +38,8 @@ ENV BASH_ENV=/opt/app-root/etc/scl_enable \
 # Also setup the 'openshift' user that is used for the build execution and for the
 # application runtime execution.
 # TODO: Use better UID and GID values
-RUN yum-config-manager --enable rhel-7-server-optional-rpms && \
+RUN yum repolist > /dev/null && \
+  yum-config-manager --enable rhel-7-server-optional-rpms && \
   INSTALL_PKGS="autoconf \
   automake \
   bsdtar \

